source/stylesheets/refills/unstyled/_vertical-tabs.scss

```diff
@@ -0,0 +1,53 @@
+.vertical-tabs-container {
+  $large-screen: em(640) !default;
+  $vertical-tabs-height: em(100);
```

Comment:
I think variables that are used for one component should remain within its scope. We gain nothing by putting them outside, beside potential conflicts with other components.